packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -74,7 +75,9 @@ class BaseAwsSdkPlugin extends Plugin {
   }
 
   isEnabled (request) {
-    return true
+    const serviceId = this.serviceIdentifier.toUpperCase()
+    const envVarValue = process.env[`DD_TRACE_AWS_SDK_${serviceId}_ENABLED`]
+    return envVarValue ? isTrue(envVarValue) : true
```

Comment:
Hi @onhate 
is there a reason why the default is set to true ?
In a case where i haven't configured  or enabled env (`DD_TRACE_AWS_SDK_${serviceId}_ENABLED`) , Ideally i wouldn't expect the traces to injected. 

I have encountered an issue where upon upgrading dd-trace lib i started seeing traces being injected into my kinesis data stream , which caused my downstream pipeline to fail.

Reply:
That was the default behavior, I introduced the env vars to be able to disable it